lib/miq_automation_engine/models/miq_ae_datastore.rb

```diff
@@ -73,7 +73,7 @@ def self.upload(fd, name = nil, domain_name = ALL_DOMAINS)
     _log.info("Upload complete (size=#{File.size(filename)})")
 
     begin
-      import_yaml_zip(filename, domain_name)
+      import_yaml_zip(filename, domain_name, User.current_tenant)
```

Comment:
@kbrock 
This is being called from the UI. The user is importing into a new domain in the datastore.